### ISSUE TITLE
Update all mentions of Ethereum Mainnet to use the wording Ethereum Mainnet

### ISSUE THREAD

**Issue by fulldecent** [ethereum/ethereum-org-website#4968]
**Describe the bug**

This is related to [EIP-2228](https://eips.ethereum.org/EIPS/eip-2228)

Currently, the ethereum.org website refers to the main network using various names. The correct and only name should be Ethereum Mainnet. With capitalization exactly like that.

**To Reproduce**

x

**Additional context**

Extensive discussion on why this is a good idea, and unanimous community consent is in that EIP.

**Want to contribute?**

Yes. Somebody with commit access please confirm your support and I will happily create such a PR.

**Comment by minimalsm** [comment]
Hey @fulldecent 

We made the transition to using Mainnet as a proper noun in Aug 2021. There will be a few that slip through the cracks, I've fixed them in #4996. Feel free to double-check yourself if there are any others that need to be changed.

I won't comment on the EIP issue as I feel it would muddy the discussion but the entire ethereum.org team supports this change and we have added this to [our style guide](https://ethereum.org/en/contributing/style-guide/#mainnet) to help enforce it going forward.
